articles/references/concepts/toc.yml

```diff
@@ -0,0 +1,4 @@
+- name: Project Layout (with Roles)
+  href: project-layout.md
+- name: Stereotypes
+  href: stereotypes.md
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.